ambry-protocol/src/test/java/com/github/ambry/protocol/RequestResponseTest.java

```diff
@@ -672,6 +672,109 @@ public void replicaMetadataRequestTest() throws IOException {
     MessageInfoAndMetadataListSerde.AUTO_VERSION = oldMessageInfoVersion;
   }
 
+  @Test
+  public void doFileCopyMetaDataRequestTest() throws IOException {
```

Comment:
Do mention in each testcase the procedure to test and what all is being checked towards the end.

Reply:
Test Case names should be self explanatory.